extensions/CocoStudio/Reader/SceneReader.h

```diff
@@ -62,8 +62,11 @@ class CC_EX_DLL SceneReader
 {
 
 public:
-	static SceneReader* sharedSceneReader();
-	static void purge();
+	CC_DEPRECATED_ATTRIBUTE static SceneReader* sharedSceneReader();
+	static SceneReader* getInstance();
+	CC_DEPRECATED_ATTRIBUTE static void purge();
+	static void destroyInstance();
+	static void destroyCocosStudio();
```

Comment:
I've just discus with ZhouNan, V3 have Cocosstudio.h file, but V2 won't have this file, and if add a new cpp file to implement this function, for both V2 and V3 need change project file for 7 platform to compile and exam running result, this may take 2 days to finish all test, Can Ccocos runtime wait 2 day for the change?